deploy_from_github_zip.bash

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+[ $# -eq 0 -o $# -gt 3 ] && {
+    echo "Usage: deploy_from_github_zip.bash <new project name> [GitHub-user/repository-name] [branch]"
+    exit 1
+}
+
+project=$1
```

Comment:
I don't think we should force the user to name their paper anything special. The PubDB can track papers and assign serial numbers in whatever way they like, but I don't think that we should impose this on the name of the directory or pdf that authors are working on.